dialogue-core/src/main/java/com/palantir/dialogue/core/QueuedChannel.java

```diff
@@ -437,4 +452,14 @@ public Timer requestQueuedTime() {
             return requestQueuedTimeSupplier.get();
         }
     }
+
+    private enum QueuedChannelTagTranslator implements TagTranslator<QueuedChannel> {
```

Comment:
I think an alternative is to pass in a `Map<String, String>` into `span().complete(...)`. I'm not actually sure what would be more performant. Using this enum is cleaner though.